src/hb-subset-plan.cc

```diff
@@ -848,6 +848,10 @@ hb_subset_plan_create_or_fail (hb_face_t	 *face,
   plan->check_success (plan->vmtx_map = hb_hashmap_create<unsigned, hb_pair_t<unsigned, int>> ());
   plan->check_success (plan->hmtx_map = hb_hashmap_create<unsigned, hb_pair_t<unsigned, int>> ());
 
+  plan->check_success (plan->name_table_overrides = hb_hashmap_create<unsigned, hb_bytes_t> ());
+  if (plan->name_table_overrides && input->name_table_overrides)
+    *plan->name_table_overrides = *input->name_table_overrides;
```

Comment:
As this is currently set up input->name_table_overrides is the owner of these strings  and plan->name_table_overrides is just holding references to those same strings. However, it's not safe to assume that the subset input object will live as long as the subset plan object (since the plan can be created separately from plan execution).

So it will be necessary to reallocate memory for each string and have the plan own it's own copy of each.

Reply:
Done.